api/specs.proto

```diff
@@ -165,6 +165,14 @@ message ContainerSpec {
 	// StopGracePeriod the grace period for stopping the container before
 	// forcefully killing the container.
 	Duration stop_grace_period = 9;
+	HealthConfig HealthCheck = 10;
+}
+
+message HealthConfig {
+	repeated string test = 1;
+	int64 	 interval = 2;
+	int64 	 timeout = 3;
```

Comment:
@runshenzhu I think `interval` and `timeout` should be durations. Can you amend your PR to update that?